app/view/templates/pages/studies/visualize/_chart_form.html

```diff
@@ -91,11 +97,6 @@ <h4 data-tooltip="{{ description }}">
           {% endfor %}
         {% endfor %}
 
-        <div class="form-row margin-top-10">
-          <a href="#" class="white-button js-select-all">Select all</a>
-          <a href="#" class="white-button flex-right js-clear-chart">Clear</a>
-        </div>
```

Comment:
I've removed the "select all" button, since it selects all visible checkboxes and that would usually just be too much. I might add a button to add/remove an entire bioreplicate, but it's a bit trickier.